apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -908,4 +957,115 @@ public void run() {
          throw new RuntimeException("After " + retryCountLimit + " retries: " + lastException);
       }
    }
+   
+   /**
+    * Gets manifest Information for DLO uploaded file
+    * @param container
+    * @param objectName
+    * @return manifest file name if DLO uploaded, else null
+    */
+   private String getManifestInfo(String container, String objectName) {
+      ObjectApi objectApi = api.getObjectApi(regionId, container);
+      Multimap<String, String> objInfo = objectApi.getWithoutBody(objectName).getHeaders();
+      if (objInfo.containsKey("X-Object-Manifest")) {
+         Collection<String> objectManifest = objInfo.get("X-Object-Manifest");
+         return String.valueOf(objectManifest.toArray()[0]);
+      }
+      return null;
+   }
+   
+   /**
+    * Gets X-Static-Large-Object for SLO uploaded file
+    * @param container
+    * @param objectName
+    * @return true if object is SLO uploaded else null
+    */
+   private String getSLOInfo(String container, String objectName) {
+      ObjectApi objectApi = api.getObjectApi(regionId, container);
+      Multimap<String, String> objInfo = objectApi.getWithoutBody(objectName).getHeaders();
+      if (objInfo.containsKey("X-Static-Large-Object")) {
+         Collection<String> objectManifest = objInfo.get("X-Static-Large-Object");
+         return String.valueOf(objectManifest.toArray()[0]);
+      }
+      return null;
+   }
+   
+   
+   /**
+    * @param containerAndPrefix
+    */
+   private void removeObjectsWithPrefix(String containerAndPrefix) {
```

Comment:
Remove this function and make the single caller explicitly call `splitContainerAndKey` and `removeObjectsWithPrefix(String, String)`.